test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -38,6 +38,7 @@
 from torch import Tensor
 from torch.nn.utils import rnn as rnn_utils
 from torch.onnx import _constants, verification
+from torch.onnx.errors import SymbolicValueError
```

Comment:
I would import module only:

```suggestion
from torch.onnx import errors
```

and use as

 errors.SymbolicValueError

Reply:
Same in torch/onnx/symbolic_opset17.py: https://github.com/pytorch/pytorch/wiki/torch.onnx-Namespacing

Reply:
Done!